edge-apps/clock/static/js/main.js

```diff
@@ -28,3 +28,5 @@ function initApp () {
 
   initDateTime()
 }
+
```

Comment:
Can you also fix things above here. `initDateTime` has bugs. Please look for a better library perhaps as the above is way too naive in the way it abbreviate months for instance and would yield "Octo" for October instead of the expected "Oct". Ideally, we'd have a framework that supports localization.

Reply:
I tested **_Moment.js_** on a browser console. Using `MMMM` will return the complete month name (and not just the first four letters). Moment.js provides [i18n](https://momentjs.com/docs/#/i18n/) out of the box. I can still continue to search for alternatives afterwards.